stdlib/public/Platform/Platform.swift

```diff
@@ -351,11 +351,6 @@ public func ioctl(
 public func fork() -> Int32 {
   fatalError("unavailable function can't be called")
 }
-
-@available(*, unavailable, message: "Please use threads or posix_spawn*()")
-public func vfork() -> Int32 {
```

Comment:
Ah, forgot that Clang defines `vfork` as a builtin, defining it to have the `returns_twice` attribute.

The loss of the curated diagnostic is still unfortunate, but probably OK.